crates/turborepo/tests/mod.rs

```diff
@@ -0,0 +1,58 @@
+use assert_cmd::Command;
```

Comment:
This file should be deleted since we won't be running these tests in CI and all of the functionality is covered by integration tests.